crates/polars-arrow/src/temporal_conversions.rs

```diff
@@ -151,7 +151,7 @@ pub fn timestamp_s_to_datetime_opt(seconds: i64) -> Option<NaiveDateTime> {
 /// converts a `i64` representing a `timestamp(ms)` to [`NaiveDateTime`]
 #[inline]
 pub fn timestamp_ms_to_datetime(v: i64) -> NaiveDateTime {
-    timestamp_ms_to_datetime_opt(v).expect("invalid or out-of-range datetime")
+    timestamp_ms_to_datetime_opt(v).expect(&format!("invalid or out-of-range datetime: {v}"))
```

Comment:
Yikes. This does a heap allocation one very function call. Which I expect are a lot.

Reply:
sorry was just debugging 😄 when I saw what looked like `np.NaT`, I figured what the issue was

Reply:
i.e.:
```
In [3]: np.datetime64('NaT').astype('int64')
Out[3]: -9223372036854775808
```